app/core/security.py

```
@@ -0,0 +1,49 @@
+    
+    to_encode.update({"exp": expire})
+    encoded_jwt = jwt.encode(to_encode, settings.SECRET_KEY, algorithm=settings.ALGORITHM)
+    
+    return encoded_jwt
+
+
+def decode_access_token(token: str) -> dict[str, Any] | None:
+    """JWT 액세스 토큰 디코딩 및 검증"""
+    try:
+        payload = jwt.decode(token, settings.SECRET_KEY, algorithms=[settings.ALGORITHM])
+        return payload
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)